dev/create-release/release-tag.sh

```diff
@@ -104,7 +109,9 @@ git commit -a -m "Preparing development version $NEXT_VERSION"
 if ! is_dry_run; then
   # Push changes
   git push origin $RELEASE_TAG
```

Comment:
Where is this tag pointing, @wangyum ?

Reply:
I'm wondering if we can have a pointer during vote like the following.
- https://github.com/apache/spark/releases/tag/v3.0.0-preview-rc2

Reply:
Yes. We still have a similar pointer. But we do not add any commits.

Reply:
😮 It's interesting. Does the pointer look like the following really?
- https://github.com/apache/spark/commit/007c873ae34f58651481ccba30e8e2ba38a692c4

Reply:
This is an example: https://github.com/wangyum/spark/releases/tag/v3.0.0-preview2-rc1

Reply:
v3.0.0-preview2-rc1: https://github.com/apache/spark/commit/1b41cb5c89e1e61480bda74c16a065413813d8ed